ui/components/ui/loading-heartbeat/index.js

```diff
@@ -11,6 +11,8 @@ export default function LoadingHeartBeat() {
   useShouldAnimateGasEstimations();
   const active = useSelector(getGasLoadingAnimationIsShowing);
 
+  if (process.env.IN_TEST) return null;
```

Comment:
Should we put this higher? I'm just thinking maybe there is no reason to run `useShouldAnimateGasEstimations` if we don't need to.

Reply:
react warns for conditionally calling hooks

Reply:
Hmm. Since `useShouldAnimateGasEstimations` calls `useGasFeeEstimates`, and since `useShouldAnimateGasEstimations` includes two `useEffect`s, it seems like it might not be worth it to put this conditional here then? Otherwise it doesn't seem like the heartbeat would be completely disabled.